Dynatrace/dynatrace-operator
# Adds operator pods log collection to support archive.

# Description

Adds collection of logs from all operator related pods into the support archive.

## How can this be tested?
* Deploy the operator
* run `kubectl -n dynatrace exec pod/$(kubectl -n dynatrace get pods | grep dynatrace-operator | cut -f1 -d' ') -- /usr/local/bin/dynatrace-operator support-archive`
* retrieve the support archive tarball (kubectl cmd will be printed on screen) and check it for log files


## Checklist
